nilearn/reporting/html_report.py

```diff
@@ -165,11 +165,11 @@ class HTMLReport(HTMLDocument):
     Methods such as save_as_html(), open_in_browser()
     are inherited from HTMLDocument
     """
-    def __init__(self, head_tpl, body):
+    def __init__(self, head_tpl, body, head_values={}):
```

Comment:
detail: maybe `head_mapping` (the parameter to `safe_substitute` is "mapping") ? or something more explicit like `head_template_parameters`